jupyverse_api/jupyverse_api/main/__init__.py

```diff
@@ -39,11 +40,21 @@ def __init__(
         app: FastAPI | str | None = None,
         host: str = "127.0.0.1",
         port: int = 8000,
+        allow_origin: Tuple = (),
         open_browser: bool = False,
         query_params: Dict[str, Any] | None = None,
         debug: bool | None = None,
         middlewares: Sequence[Callable[..., ASGI3Application] | dict[str, Any]] = (),
     ) -> None:
+        if allow_origin:
```

Comment:
At this point, allow_origin is a string, not a tuple. Should it be cast to a tuple before configuring the middleware?